yellowbrick/gridsearch/pcolor.py

```diff
@@ -0,0 +1,225 @@
+"""
+Colorplot visualizer for gridsearch results.
+"""
+
+import time
+import numpy as np
+
+from .base import GridSearchVisualizer
+from ..exceptions import YellowbrickValueError
+
+from sklearn.metrics.pairwise import pairwise_distances
+from sklearn.preprocessing import LabelEncoder
+from matplotlib.mlab import griddata
+
+
+## Packages for export
+__all__ = [
+    "GridSearchColorPlot",
+    "gridsearch_color_plot"
+]
+
+
+##########################################################################
+## Dimension reduction utility
+##########################################################################
+
+def param_projection(cv_results, param_1, param_2):
+    """
+    Projects the grid search results onto 2 dimensions.
+
+    The display value is taken as the max over the non-displayed dimensions.
+
+    Parameters
+    ----------
+    cv_results : dict
+        A dictionary of results from the `GridSearchCV` object's `cv_results_`
+        attribute
+
+    param_1 : string
+        The name of the parameter to be visualized on the horizontal axis.
+
+    param_2 : string
+        The name of the parameter to be visualized on the vertical axis.
+
+    Returns
+    -------
+    ax : matplotlib axes
+        Returns the axes that the classification report was drawn on.
+    """
+    # Get unique values of the two display parameters
+    x_vals = sorted(list(set(cv_results['param_' + param_1].compressed())))
+    y_vals = sorted(list(set(cv_results['param_' + param_2].compressed())))
+    n_x = len(x_vals)
+    n_y = len(y_vals)
+
+    # Get mapping from parameter value -> integer index
+    int_mapping_1 = {value: idx for idx, value in enumerate(x_vals)}
+    int_mapping_2 = {value: idx for idx, value in enumerate(y_vals)}
+
+    # Translate each gridsearch result to indices on the grid
+    idx_x = [int_mapping_1[value] if value else None
+             for value in cv_results['param_' + param_1]]
+    idx_y = [int_mapping_2[value] if value else None
+             for value in cv_results['param_' + param_2]]
+
+    # Create an array of all scores for each value of the display parameters.
+    # This is a n_x by n_y array of lists with `None` in place of empties
+    # (my kingdom for a dataframe...)
+    all_scores = [[None for _ in range(n_x)] for _ in range(n_y)]
+    for x, y, score in zip(idx_x, idx_y, cv_results['mean_test_score']):
```

Comment:
It feels like we should allow the user to choose which metric as an input to the visualizer (which defaults to `mean_test_score` but could also be - `mean_train_score`, `mean_fit_time`, and `mean_score_time`.